helm/cluster/templates/clusterapi/controlplane/_helpers.tpl

```diff
@@ -0,0 +1,8 @@
+{{- define "cluster.internal.test.controlPlane.machineTemplate.spec" -}}
```

Comment:
Is the `test` intended?

Reply:
Yes, this is just a "dummy" helper that is used in the CI.